GitFlowTraining/Credit.cs

```diff
@@ -16,17 +16,46 @@ public Credit(decimal amount)
 
         public int CompareTo(Credit c)
         {
-            throw new NotImplementedException();
+            if (c != null)
+            {
+                if (c.Amount > Amount)
+                {
+                    return -1;
+                }
+                else if (c.Amount < Amount)
+                {
+                    return 1;
+                }
+                else
+                {
+                    return 0;
+                }
+            }
+            else
+            {
+                throw new ArgumentException();
+            }
```

Comment:
Good job, I strongly recommend you to place the logic with Argument exception on the top
```cs
public int CompareTo(Credit c)
{
  if (c == null) 
  {
      throw new ArgumentException();
  }
// other logic...
```

It is a good practice to write validation logic at beginning of the method...